src/ws/cockpitauth.c

```diff
@@ -49,9 +49,12 @@
 #define ACTION_NONE "none"
 #define LOCAL_SESSION "local-session"
 
+/* for the time being, we only support running an installed cockpit-bridge on the remote,
+ * and leave beibooting to the flatpak */
```

Comment:
I would say "leave beipacking to the flatpak"... if I understand things correctly. We are clearly using beiboot on the next line, but what we don't do is use `--remote-bridge=auto`.

Reply:
The terminology is indeed a bit confusing -- the bei*pack*ing technically already happens at build time (building the bridge .xz). "beibooting" is both the process of sending the beipack to a remote end and running it, and also the name of the program (cockpit.beiboot). After this PR lands, I'd actually like to rename it "back" to "cockpit.ssh", but I thought it would be too confusing to do that here already (as there's also the C version of cockpit-ssh).

This comment hopefully won't live long -- next step after this PR is to enable beibooting for the cockpit/ws container (in PR #21042), and then also from regular rpm/deb packages (tracked in jira already, and I think I have a solution -- just can't look up the number right now,  not enough internet in the train).

Reply:
Ah! The thing that moves your beiboot is called "davit", so we could have cockpit.davit... :-) But cockpit.ssh sounds better!